app/scripts/views/sign_up.js

```diff
@@ -183,6 +183,8 @@ define(function (require, exports, module) {
         signinUri: this.broker.transformLink('/signin')
       };
 
+      this.setShowSyncSuggestion(context);
```

Comment:
deleting the setShowSyncSuggestion function and creating a context function will erase the context from the original view